<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DonationImportForm</class>
 <widget class="QWidget" name="DonationImportForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../PiTres.qrc">
    <normaloff>:/icons/resources/icons/goldbars.png</normaloff>:/icons/resources/icons/goldbars.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../PiTres.qrc">:/icons/resources/icons/goldbars.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Donation Wizard&lt;/span&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>CSV-Import</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QStackedWidget" name="stackedImport">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="pageInit">
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Select File to import:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="editImportFile"/>
             </item>
             <item>
              <widget class="QPushButton" name="btnChooseDonationImport">
               <property name="text">
                <string>Choose</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Use Separator:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="importSeparator">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>;</string>
             </property>
             <property name="maxLength">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkFirstLineHeader">
             <property name="text">
              <string>Select if the first line is a Header-Row</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageFileImport">
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Value in Column</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="importColumnMaster">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>9999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>complies with one of</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="importColumnMemberfields">
               <item>
                <property name="text">
                 <string>All Fields</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>UID, Name, Surname, Company</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Telephone numbers</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>EMail Addresses</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Search-Value match</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="importRegexMatch">
               <property name="text">
                <string>.*(\d+).*</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>is replaced by</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="importRegexReplace">
               <property name="text">
                <string>00\1</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>Amount is in Column</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="importAmountColumn">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>9999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Section is in Column</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="importSectionColumn">
               <property name="maximum">
                <number>9999</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tablePreviewImport"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_4">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QGroupBox" name="importMapSections">
             <property name="title">
              <string>Map found Sections with existent ones</string>
             </property>
             <layout class="QFormLayout" name="formLayout_4">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::ExpandingFieldsGrow</enum>
              </property>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTableWidget" name="tableWidget"/>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Additional information</string>
             </property>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Objective:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="editObjectiveImport">
                <property name="placeholderText">
                 <string>Choose or enter a new Objective this donation is made for</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Booking Memo:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="editMemoImport">
                <property name="placeholderText">
                 <string>Give some text for the Booking</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageProgress">
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Importing %1 Donations:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressImport">
             <property name="format">
              <string>%p%</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnImportBack">
           <property name="text">
            <string>Back</string>
           </property>
           <property name="icon">
            <iconset resource="../PiTres.qrc">
             <normaloff>:/icons/resources/icons/expense.png</normaloff>:/icons/resources/icons/expense.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnImportNext">
           <property name="text">
            <string>Next</string>
           </property>
           <property name="icon">
            <iconset resource="../PiTres.qrc">
             <normaloff>:/icons/resources/icons/income.png</normaloff>:/icons/resources/icons/income.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnImportSave">
           <property name="text">
            <string>Import</string>
           </property>
           <property name="icon">
            <iconset resource="../PiTres.qrc">
             <normaloff>:/icons/resources/icons/disk_blue.png</normaloff>:/icons/resources/icons/disk_blue.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Manually record</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackedRecord">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Record a Donation</string>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::ExpandingFieldsGrow</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Person, Member:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="editMember">
                <property name="placeholderText">
                 <string>Select an existing Member or fill in all values manually</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Company:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="editCompany"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Address:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="editAddress">
                <property name="placeholderText">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>Zip, City, Country:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,1">
                <item>
                 <widget class="QLineEdit" name="editCity"/>
                </item>
                <item>
                 <widget class="QLineEdit" name="editCountry">
                  <property name="maxLength">
                   <number>2</number>
                  </property>
                  <property name="placeholderText">
                   <string>2 Char Code</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Telephone:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="editPhone"/>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Mobile Number:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="editMobile"/>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>EMail Address:</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QLineEdit" name="editEmail"/>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Amount:</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QDoubleSpinBox" name="editAmount">
                <property name="maximum">
                 <double>99999.990000000005239</double>
                </property>
               </widget>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Objective:</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <widget class="QLineEdit" name="editObjectiveRecord">
                <property name="placeholderText">
                 <string>Choose or enter a new Objective this donation is made for</string>
                </property>
               </widget>
              </item>
              <item row="10" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Booking Memo:</string>
                </property>
               </widget>
              </item>
              <item row="10" column="1">
               <widget class="QLineEdit" name="editMemoRecord">
                <property name="placeholderText">
                 <string>Give some text for the Booking</string>
                </property>
               </widget>
              </item>
              <item row="11" column="1">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>133</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>For Section:</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QComboBox" name="recordSectionCombo"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="recordStatusPage">
             <property name="title">
              <string>Status</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnRecordPreview">
           <property name="text">
            <string>Preview</string>
           </property>
           <property name="icon">
            <iconset resource="../PiTres.qrc">
             <normaloff>:/icons/resources/icons/disk_blue.png</normaloff>:/icons/resources/icons/disk_blue.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRecordImport">
           <property name="text">
            <string>Record Donation</string>
           </property>
           <property name="icon">
            <iconset resource="../PiTres.qrc">
             <normaloff>:/icons/resources/icons/disk_blue.png</normaloff>:/icons/resources/icons/disk_blue.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionReloadImportfile">
   <property name="text">
    <string>Reload Importfile</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../PiTres.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>importRegexReplace</sender>
   <signal>textChanged(QString)</signal>
   <receiver>actionReloadImportfile</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importRegexMatch</sender>
   <signal>textChanged(QString)</signal>
   <receiver>actionReloadImportfile</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importColumnMaster</sender>
   <signal>valueChanged(int)</signal>
   <receiver>actionReloadImportfile</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importSectionColumn</sender>
   <signal>valueChanged(int)</signal>
   <receiver>actionReloadImportfile</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>371</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
